README.md

```diff
@@ -179,6 +179,15 @@ Branch returns explicit parameters every time. Here is a list, and a description
 | +clicked_branch_link | Denotes whether or not the user clicked a Branch link that triggered this session
 | +click_timestamp | Epoch timestamp of when the click occurred
 
+##### Initialization tips and tricks
+
+Very important thing is to call Branch.InitSession(...) at the start of your app else Branch has not time to registry callback and you will receive nothing.
```

Comment:
Is there a certain place in the Unity project where you need to call it so that we can help other teams avoid this mistake? It looks like you didn't update the initSession location in the testbed.